Framework/PythonInterface/test/python/mantid/kernel/VisibleWhenPropertyTest.py

```diff
@@ -1,17 +1,26 @@
 from __future__ import (absolute_import, division, print_function)
 
 import unittest
-from mantid.kernel import VisibleWhenProperty, PropertyCriterion
+from mantid.kernel import VisibleWhenProperty, PropertyCriterion, LogicOperator
+
 
 class VisibleWhenPropertyTest(unittest.TestCase):
 
     def test_construction_with_name_criterion_only_succeeds(self):
         p = VisibleWhenProperty("OtherProperty", PropertyCriterion.IsDefault)
+        self.assertIsNotNone(p)
 
     def test_construction_with_name_criterion_value_succeeds(self):
         p = VisibleWhenProperty("OtherProperty", PropertyCriterion.IsEqualTo, "value")
+        self.assertIsNotNone(p)
+
+    def test_multiple_condition_constructio(self):
+        a = VisibleWhenProperty("PropA", PropertyCriterion.IsDefault)
+        b = VisibleWhenProperty("PropB", PropertyCriterion.IsDefault)
+        result = VisibleWhenProperty(a, b, LogicOperator.And)
+        self.assertIsNotNone(result)
```

Comment:
We can't test from Python if these actually work as we would need to subscribe an algorithm and manipulate the properties. 
Instead the Python tests just check we can construct it and the C++ unit tests test all combinations of operators work. 
I'll add this note to the unit tests on the Python side